c-sharp.md

```diff
@@ -5,70 +5,37 @@ tags = ["c#", "csharp", "dotnet", ".net", "webassembly"]
 template = "page_lang"
 [extra]
 author = "Fermyon Staff"
-last_modified = "2022-03-10T21:50:50Z"
+last_modified = "2023-10-26T00:50:50Z"
 ---
 # C# and .NET in WebAssembly
 
 The .NET (aka dotnet) ecosystem supports a variety of languages, including C# (C sharp) and ASP.NET. Programs that target .NET are compiled to an intermediary bytecode format that runs in the .NET Common Language Runtime (CLR). This makes .NET an example of a [virtual machine language](/blog/scripts-vs.compiled-wasm.md).
 
 ## Available Implementations
 
-The .NET ecosystem has long had support for browser-side WebAssembly via the [Blazor toolkit](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor).
+The .NET ecosystem has long had support for browser-side WebAssembly via the [Blazor toolkit](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor). As of .NET 7, it has support for WebAssembly as a compile target.
 
-The team at Microsoft is rapidly building out a number of WebAssembly tools. In 2022, it seems reasonable to expect that Microsoft will have new tooling for WebAssembly. C# compiled using the .NET toolchain will have access to these WebAssembly features.
-
-At the time of this writing, the closest we have seen to a full WASI runtime is Steve Sanderson's example [.NET WASI Runtime](https://github.com/SteveSandersonMS/dotnet-wasi-runtime).
+The team at Microsoft is rapidly building out a number of WebAssembly tools. .NET 7 includes support for compiling to WebAssembly. Read [Ivan Towlson's excellent tutorial](https://www.fermyon.com/blog/dotnet-wasi) on building a WebAssembly app with .NET.
```

Comment:
The instructions in that are, I think, quite out of date with the .NET 8 changes.

Somebody should write an updated post.  And you should be unreasonably nice about that one too.